packages/pages/src/components/analytics/analytics.test.tsx

```diff
@@ -10,13 +10,31 @@ import { AnalyticsProvider } from "./provider.js";
 import { useAnalytics } from "./hooks.js";
 import { AnalyticsScopeProvider } from "./scope.js";
 
+jest.mock("../../util/runtime.js", () => {
+  const runtime = {
+    name: "browser",
+    isServerSide: false,
+    version:
+      "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36",
+  };
+  return {
+    getRuntime: () => runtime,
+  };
+});
+
 // The following section of mocks just exists to supress an error that occurs
 // because jest does not implement a window.location.navigate.  See:
 // https://www.benmvp.com/blog/mocking-window-location-methods-jest-jsdom/
 // for details.
 const oldWindowLocation = global.location;
 
 beforeAll(() => {
+  global.process = {
```

Comment:
Would it make sense to add an isDevelopment flag to runtime, since there are eccentricities with using process directly? Also little easier to mock lol

Reply:
isDevelopment as in when running tests on jest, or as in running the development server?

Reply:
Adding an isDevelopment to the Runtime class (the latter)

Reply:
Certain parts of pagesjs are expected to run server side in node, deno, or both. Passing in that flag would mean we'd have to figure out the runtime outside of the runtime class